eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -1182,6 +1183,21 @@ votingSpec =
         passNEpochs 2
         -- DReps voted yes too for the hard-fork, so protocol version is incremented
         getProtVer `shouldReturn` nextProtVer
+      it "Can hard-fork to PV 11" $ do
```

Comment:
This test belongs in EnactSpec instead of RatifySpec

Also it would be better if you could extend this test: https://github.com/IntersectMBO/cardano-ledger/blob/d90eb4df4651970972d860e95f1a3697a3de8977/eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/EnactSpec.hs#L213

instead of manually setting protocol version to `10`

Reply:
Nevermind. We run all tests for every protocol version in the era: https://github.com/IntersectMBO/cardano-ledger/blob/d90eb4df4651970972d860e95f1a3697a3de8977/eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp.hs#L96

Reply:
Let's remove this test completely